setup.py

```diff
@@ -56,8 +56,9 @@
             # AT2L0 and its simulator:
             'ioc-lfe-at2l0-calc=solid_attenuator.ioc_lfe_at2l0_calc.__main__:main',  # noqa
             'ioc-sim-at2l0=solid_attenuator.ioc_sim_at2l0.__main__:main',
-            # SXR solid attenuator entrypoint, including AT2K2:
+            # SXR solid attenuator entrypoint, including AT2K2 and AT1K2:
```

Comment:
Yours uses a different entrypoint. But at1k4 uses it, so let's fix this:
```suggestion
            # SXR solid attenuator entrypoint, including AT2K2 and AT1K4:
```

Reply:
@wwright-slac you forgot to fix this...

Reply:
Ah, misread. One moment

Reply:
@klauer Should this instead say 'including AT1K2 and AT1K4'?

Reply:
So AT2K2 and AT1K4 use the same entrypoint since they're the 4 blade SXR solid attenuators
AT1K2 uses its own entrypoint that you put together - `ioc-kfe-at1k2-calc`
Separately, AT2L0 has its own as well

Same page?

Reply:
Ah, I see. I was reading the comment as "this applies to everything below this, until the next comment" instead of just the next line